lib/tasks/bulk_change_applications_to_pending.rake

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require "rake"
+
+namespace :one_off do
+  # Bulk change NPQ applications to pending.
+  # Accepts a CSV of application (ecf) IDs (without a header row) and a dry run
+  # flag. If the dry run flag is true, or not set, the changes will not be performed
+  # but the changes that would be made will be logged. Set dry run to false
+  # to commit the changes.
+  #
+  # Example usage (dry run):
+  # bundle exec rake 'one_off:bulk_change_to_pending[applications.csv]'
+  #
+  # Example usage (perform change):
+  # bundle exec rake 'one_off:bulk_change_to_pending[applications.csv,false]'
+  desc "Change NPQ applications to pending"
+  task :bulk_change_to_pending, %i[file dry_run] => :environment do |_task, args|
+    logger = Logger.new($stdout)
+    csv_file_path = args[:file]
+    dry_run = args[:dry_run] != "false"
+    unless File.exist?(csv_file_path)
+      logger.error "File not found: #{csv_file_path}"
+      return
+    end
+
+    application_ecf_ids = CSV.read(csv_file_path, headers: false).flatten
+
+    logger.info "Bulk changing #{application_ecf_ids.size} applications to pending#{' (dry run)' if dry_run}..."
+
```

Comment:
`sleep(10)` - gives the developer time to think 'oops' and press ctrl-c ? Might be belt and braces though, there's already a dry run feature? thoughts?

Reply:
I think this one might be overkill.
I'd expect the developer will already run this with dry-run first, and the effort in creating/uploading the CSV file in the first place means it shouldn't be a trivial thing to run accidentally.